courts_db/data/courts.json

```diff
@@ -18742,7 +18808,11 @@
         "name": "Supreme Court of New Hampshire",
         "regex": [
             "Supreme Court of New Hampshire",
-            "New Hampshire Supreme Court"
+            "New Hampshire Supreme Court",
+            "Supreme Court of New Hampshire FROM CARROLL CIRCUIT COURT",
+            "Supreme Court of New Hampshire(\\.|,)? (Auburn|Berlin|Claremont|Colebrook|Concord|Derry|Dover|Durham|Exeter|Franklin|Goffstown|Gorham|Greenville|Hampton|Hanover|Haverhill|Henniker|Hillsborough|Hooksett|Jaffrey|Keene|Laconia|Lancaster|Lebanon|Manchester|Merrimack|Milford|Nashua|Newport|New London|Ossipee|Peterborough|Pittsfield|Plaistow|Plymouth|Portsmouth|Rochester|Salem|U.S.|Wilton|) District Court",
+            "Supreme Court of New Hampshire(\\.|,)? (Amherst|Concord|Conway|Dover|Durham|Epping|Exeter|Farmington|Greenville|Hampton|Jaffrey|Keene|Laconia|Lebanon|Manchester|Milford|Nashua|Pelham|Peterborough|Pittsfield|Portsmouth|Rye|Wilton|Wolfeboro) Municipal Court",
+            "Supreme Court of New Hampshire(\\.|,)? Municipal Court of (Amherst|Berlin|Charlestown|Concord|Conway|Dover|Exeter|Farmington|Greenville|Hampton|Jaffrey|Keene|Laconia|Lebanon|Manchester|Milford|Nashua|Portsmouth|Rye|Wilton|Wolfeboro)"
```

Comment:
this should probably be broken up into its child courts- and parent courts - now that we have a mechanism for it.